app/renderer/js/utils/domain-util.js

```diff
@@ -101,6 +101,48 @@ class DomainUtil {
 		return false;
 	}
 
+	checkCertError(domain, serverConf, error, silent) {
+		return new Promise((resolve, reject) => {
+			if (silent) {
+				// since getting server settings has already failed
+				resolve(serverConf);
+			} else {
+				// Report error to sentry to get idea of possible certificate errors
+				// users get when adding the servers
+				logger.reportSentry(new Error(error));
+				const certErrorMessage = `Do you trust certificate from ${domain}? \n ${error}`;
+				const certErrorDetail = `The organization you're connecting to is either someone impersonating the Zulip server you entered, or the server you're trying to connect to is configured in an insecure way.
+				\nIf you have a valid certificate please add it from Settings>Organizations and try to add the organization again.
+				\nUnless you have a good reason to believe otherwise, you should not proceed.
+				\nYou can click here if you'd like to proceed with the connection.`;
```

Comment:
Why is there a need to use template string literals and `\n`? This will mess up the formatting by adding double new lines.

Reply:
Also, I suggest you don't use template literal here since the indentation will show up. They are generally best for multiline string on top level.

Reply:
> Why is there a need to use template string literals and \n?

I can't answer that - I just copied over the messages from the old method. 
I'll change this in a separate commit.

Reply:
Ah, can you try seeing if it affects the formatting of the dialog that shows up?

Reply:
No, there's no change. 

Here's the dialog generated by the PR:

![mine](https://user-images.githubusercontent.com/24617297/58439700-28b6fd00-80f3-11e9-93df-c9c25816ed32.png)

Here's `master`:

![Screenshot from 2019-05-28 02-47-49](https://user-images.githubusercontent.com/24617297/58439709-2fde0b00-80f3-11e9-9877-7e0993b17a63.png)

The difference in the error message is because of the change in the first `request` made.

Reply:
Cool! The indentation will show up if you don't use `\n`.